crux_kv/src/value.rs

```diff
@@ -1,37 +0,0 @@
-use serde::{Deserialize, Serialize};
-
-/// The value stored under a key.
-///
-/// `Value::None` is used to represent the absence of a value.
-///
-/// Note: we can't use `Option` here because generics are not currently
-/// supported across the FFI boundary, when using the builtin typegen.
-#[derive(Clone, Serialize, Deserialize, Debug, PartialEq, Eq)]
-pub enum Value {
-    None,
-    Bytes(#[serde(with = "serde_bytes")] Vec<u8>),
```

Comment:
This was important for two reasons: 

1. signalling not found
2. serializing bytes better with formats that have special handling for them.